Alex Cohen Project Code/Populates.sql

```diff
@@ -0,0 +1,18 @@
+-- Alex Cohen Husky On Call Scheduler Project SQL Code
+-- Populate the following look-up tables: tblSTATUS, tblSHIFT_TYPE
+INSERT INTO tblSTATUS(StatusTitle, StatusDescr)
```

Comment:
For the sake of our grade, make sure you have sufficient rows in this table :) I heard it might be minimum 5 but I am not sure? If you need an extra status to reach 5, you can make something up i.e. `('Cancelled', 'Shift no longer needs to be worked')`